SELF-ASSESMENT.md

```diff
@@ -44,3 +56,359 @@ See [README](https://github.com/jaegertracing/jaeger/#jaeger---a-distributed-tra
+
+* When the services are instrumented with the Jaeger tracer, Jaeger's client libraries by default produce and send traces to Jaeger.
+* These traces are accessible through the Jaeger Query User Interface and are kept in Jaeger's storage backend (Cassandra, Elasticsearch).
+
+#### Cloud Native storage option:
+
+Jaeger is compatible with a number of storage backends that can be used to store and analyze traces in a cloud-native environment, including Cassandra, Elasticsearch, and Kafka.
+
+* Features and benefits that are specific to each storage backend include cost-effectiveness, scalability, and high availability.
+* Trace volume, retention needs, and infrastructure already in place are a few of the factors that must be taken into consideration while selecting a storage backend.
+
+#### Kubernetes Integration:
+
+Jaeger's Helm chart allows it to be deployed in Kubernetes.
+
+* The Helm chart offers flexibility in setting up different Kubernetes resources, including services, deployments, and configmaps, as well as in configuring Jaeger's components, including the choice of storage backend.
+* Jaeger can now be easily deployed and managed in a cloud-native environment, and Kubernetes' sophisticated features, such as self-healing and rolling upgrades, may be utilized.
+
+#### Prometheus Integration:
+
+Prometheus is a well-liked open-source monitoring and alerting toolkit that can be combined with Jaeger.
+
+* Metrics data and traces can be correlated by merging Jaeger and Prometheus. For instance, you can investigate the related traces in Jaeger to find the source of a latency spike that you see in Prometheus.
+* Additionally, you may make custom dashboards to keep an eye on your microservices in a cloud-native environment by utilizing Jaeger's interface with Grafana, a potent data visualization and monitoring tool.
+* Jaeger excels in distributed trace capture, while Prometheus focuses on time-series metrics for system monitoring. Together, they provide a comprehensive view of distributed system behavior and performance.
+
+## Security issue resolution
+
+#### Responsible Disclosures Process.
+
+_A outline of the project's responsible disclosures process should suspected security issues, incidents, or vulnerabilities be discovered both external and internal to the project. The outline should discuss communication methods/strategies._
+
+Vulnerabilities are shared with users and then publicly.
+
+#### Vulnerability Response Process.
+
+_Who is responsible for responding to a report. What is the reporting process? How would you respond?_
+
+A report can be sent via email at jaeger-tracing@googlegroups.com, open an issue on GitHub or send a message to the #jaeger channel on the CNCF Slack encrypted with Jaeger’s public key. Responses can be sent directly via the method used to contact the maintainers by the maintainers.
```

Comment:
incorrect. Official process is in the docs, there is no need to duplicate it here https://www.jaegertracing.io/report-security-issue/